devtools/create_bladerunner/subtitles/mixResourceCreator/packBladeRunnerMIXFromPCTLKXLS04.py

```diff
@@ -252,14 +252,14 @@ def initOverrideEncoding(pathToConfigureFontsTranslationTxt):
 									and fontCateg_targetEnc_OOOGlyphs_Tuple[1] != ''):
 								tmpFontCateg = fontCateg_targetEnc_OOOGlyphs_Tuple[0]
 								tmpTargetEncodingForThisFont = fontCateg_targetEnc_OOOGlyphs_Tuple[1]
-								if ( tmpFontCateg not in zip(*DEFAULT_TARGET_ENCODING_PER_FONT)[0]):
+								if ( tmpFontCateg not in list(zip(*DEFAULT_TARGET_ENCODING_PER_FONT))[0]):
 									print ('[Error] Invalid Font name specified in configureFontsTranslation text file!')
 									print ('        Valid values are: ', ", ".join( zip(*DEFAULT_TARGET_ENCODING_PER_FONT)[0] ))
```

Comment:
You are missing the same kind of fix here.
One small optimization: you can replace `list(zip(*...))[0]` by `next(zip(*...))`.
This avoids building the full list to take only the first element.

Reply:
Done. I agree that we should drop python2 support but @antoniou79 is the author and maintainter of this code so I think it's his decisition